src/services/help-modal-service/help-texts/usertask-usage.ts

```diff
@@ -0,0 +1,43 @@
+import {HelpText} from '../../../contracts/index';
+
+export const UserTaskUsage: HelpText = {
+  title: 'How to use UserTasks',
+  message: `To use UserTasks you must add one or more form fields to the task.
+
+    The form fields have different types such as:
+
+    - Text (shows an input field for texts)
+    - Integer (shows an input field for numbers)
+    - Decimal (shows an input field for decimal number)
+    - Truth value (shows a checkbox, especially if the UserTask is a "confirm" control)
+    - Date (shows an input field for dates)
+    - Enumeration (shows a dropdown list)
+    - Custom
+
+    Each form field also has the following properties:
+
+    - ID (the ID of the form field used for the token)
```

Comment:
Das "used for the token" find ich hier wenig hilfreich. 
Was sagt mir das als Endbenutzer?

Und auch wie weiter oben angemerkt, würde ich hier auf Klammern verzichten.

```suggestion
    - ID: The ID of the form field
```

Reply:
Ich hätte das hoffentlich als Benutzer so verstanden dass ich nur über diese ID später im Token auf das Formfield zugreifen kann.

Reply:
Dann würde ich das vllt so formulieren:

```suggestion
    - ID: The form field's ID. Can be used to access the form field through the token
```